File: multiTable.js
/** This function takes in a number and prints out a multiplication table
of the number up to 10 */

const printTable = (num) => {
	let table = "";
	for (let i = 1; i < 11; i++) {
		table +=
		`${i} * ${num} = ${i * num}${i==10?'':'\n'}`
	}
	return table
}

console.log(printTable(3))

	

File: multi_table.py
## Take number and create multiplication table to 10 with number

def multi_table(num):
	table = ''
	for i in range(1,11):
		if i == 10:
			table += str(i) + " * " + str(num) + " = " + str((i*num))
		else:
		
			table += str(i) + " * " + str(num) + " = " + str((i*num)) + '\n'
	return table

print(multi_table(3))

